package.json

```diff
@@ -142,7 +142,7 @@
     "mixin-deep": "^1.3.2",
     "set-value": "^2.0.1",
     "js-yaml": "^3.13.1",
-    "**/handlebars": "4.5.3",
+    "handlebars": "4.7.7",
```

Comment:
`handlebars` was previously used by `istanbul-reports@^2.2.4` but with the updated version `istanbul-reports@^3.0.2` the dependency is no more required, so is it possible to remove `handlebars` entirely?